test/libsolidity/semanticTests/inlineAssembly/mcopy_memory_expansion.sol

```diff
@@ -0,0 +1,36 @@
+contract C {
+    function expansion_on_write_only() public returns (uint newMsize) {
+        assembly {
+            mcopy(0xfff0, 0, 1)
+            newMsize := msize()
+        }
+    }
+
+    function expansion_on_read_only() public returns (uint newMsize) {
+        assembly {
+            mcopy(0, 0xfff0, 1)
+            newMsize := msize()
+        }
+    }
```

Comment:
Not sure what to do about this test. It won't pass `soltest_all` because you cannot use `msize()` with optimizer enabled. I could easily add an option to skip a semantic test when optimizer is enabled, but the fact that we don't have such an obvious option already makes me think that there could be some reason behind it.

On the other hand, this is really testing evmone's behavior rather than anything substantial in our implementation so maybe we're fine without it.

Reply:
For now I added a fixup that removes the test.